### ISSUE TITLE
Could this loss function be used for mutil classification？

### ISSUE THREAD

**Issue by kuro-z** [Alibaba-MIIL/ASL#51]
Could this loss function be used for mutil classification？

**Comment by mrT23** [comment]
yes

**Comment by kuro-z** [comment]
I can't use it for multiclass networks. Error reported in loss.backward().

**Comment by kuro-z** [comment]
I know the reason because of label smoothing.
